src/cow-react/modules/limitOrders/utils/getAddress.ts

```diff
@@ -1,7 +1,7 @@
 import { Currency } from '@uniswap/sdk-core'
 import { WrappedTokenInfo } from 'state/lists/wrappedTokenInfo'
 
-export function getAddress(currency: Currency | null): string | null {
+export function getAddress(currency: Currency | null | undefined): string | null {
```

Comment:
why not making `currency` optional if we really accept undefined?

Reply:
<img width="440" alt="image" src="https://user-images.githubusercontent.com/7122625/202666699-6cd52910-54b9-4065-b7f5-a41ea0095959.png">

There is a small difference :)
If we specify an argument as optional `(param?: string)` then we can call the function without passing the parameter.
And if we specify an argument as `undefinedable` then we cannot call the function without passing the parameter.

Reply:
So, I wouldn't get an opportunity to call getAddress() without arguments